flexiprep/src/main/resources/nl/lumc/sasc/biopet/pipelines/flexiprep/flexiprepReadSummary.ssp

```diff
@@ -1,14 +1,8 @@
-#import(nl.lumc.sasc.biopet.utils.summary.db.SummaryDb)
 #import(nl.lumc.sasc.biopet.utils.summary.db.Schema.Sample)
 #import(nl.lumc.sasc.biopet.utils.summary.db.Schema.Library)
-#import(nl.lumc.sasc.biopet.utils.summary.db.SummaryDb._)
-#import(nl.lumc.sasc.biopet.utils.summary.db.SummaryDb.Implicts._)
 #import(nl.lumc.sasc.biopet.core.report.ReportPage)
-#import(scala.concurrent.Await)
-#import(scala.concurrent.duration.Duration)
 #import(java.io.File)
 #import(nl.lumc.sasc.biopet.pipelines.flexiprep.FlexiprepReport)
-<%@ var summary: SummaryDb %>
```

Comment:
This is only place where a line like this is removed: `<%@ var summary: SummaryDb %>`. I would expect to see commits like this much more, this mean SummaryDb is still used in a lot of places where it's not needed anymore.